src/parser/utils.ts

```diff
@@ -20,5 +20,8 @@ export const getPathWithExtension = (pathWithoutExtension: string): string => {
     if (fs.existsSync(pathWithoutExtension + '.ts')) {
         return pathWithoutExtension + '.ts';
     }
-    throw new Error("Can't find file at " + pathWithoutExtension);
+    // NOTE(john): This is a bit of a hack, but it's necessary for when we
+    // have a file that doesn't have an extension. This will happen when we
+    // delete all of the type files before re-running graphql-flow again.
```

Comment:
**question:** What is the side-effect of doing this? Can you elaborate in the comment on why this will "just work"? Or perhaps add a comment in the `parse.ts` code that explains what an empty string means?